WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -1239,6 +1239,9 @@ - (void)handleDefaultAccountChangedNotification:(NSNotification *)notification
         }];
     } else {
         // No need to check for welcome screen unless we are signing out
```

Comment:
I know this is not part of your PR, but do you know what this comment means?  It sounds to me like it's either outdated or doesn't have enough details to be of value.  Thoughts?